### PR TITLE
Update with latest changes on platform.

### PR DESCRIPTION
**Pull request by benpatterson**

### CODE REVIEW

File: playbooks/roles/test_build_server/files/test-development-environment.sh [hunk 1]
```diff
@@ -18,11 +18,11 @@ cd edx-platform-clone
 
 # This will run all of the setup it usually runs, but none of the
 # tests because TEST_SUITE isn't defined.
-source scripts/all-tests.sh
+source scripts/jenkins-common.sh
```
Comment (benpatterson) [inline playbooks/roles/test_build_server/files/test-development-environment.sh:21]: This is simply old and needed an update

File: playbooks/roles/test_build_server/files/test-development-environment.sh [hunk 2]
```diff
@@ -18,11 +18,11 @@ cd edx-platform-clone
...
 
 # Now we can run a subset of the tests via paver.
 # Run some of the common/lib unit tests
-paver test_lib -t common/lib/xmodule/xmodule/tests/test_stringify.py
+# paver test_lib -t common/lib/xmodule/xmodule/tests/test_stringify.py
```
Comment (benpatterson) [inline playbooks/roles/test_build_server/files/test-development-environment.sh]: This test no longer works this way (get a DJANGO_SETTINGS error).

**Comment by benpatterson** [comment]
@clytwynec @jzoldak 

Passing AMI build here: https://build.testeng.edx.org/view/All/job/build-jenkins-worker-ami/5/

File: playbooks/roles/test_build_server/files/test-development-environment.sh (hunk 2 above)
Comment (clytwynec) [inline playbooks/roles/test_build_server/files/test-development-environment.sh]: FYI: There is a ticket in progress for fixing the issue with this test. ([SOL570](https://openedx.atlassian.net/browse/SOL-570))

**Comment by clytwynec** [comment]
:+1:

File: playbooks/roles/test_build_server/files/test-development-environment.sh (hunk 2 above)
Comment (jzoldak) [inline playbooks/roles/test_build_server/files/test-development-environment.sh]: We should put a TODO with the ticket number above the commented out line.

**Comment by jzoldak** [comment]
:+1:

File: playbooks/roles/test_build_server/files/test-development-environment.sh (hunk 2 above)
Comment (benpatterson) [inline playbooks/roles/test_build_server/files/test-development-environment.sh]: Done.

**Comment by benpatterson** [comment]
Updated. Merging.

**Pull request merged**